### PR TITLE
Upgraded buildTools version, gradle wrapper version, disabled com.nee…

### PR DESCRIPTION
**Pull request by rcd27**
…nbedankt.android-apt for AndroidStudio v3.x

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,7 +43,7 @@ allprojects {
         jcenter()
     }
 
-    ext.buildToolsVer = '25.0.1'
+    ext.buildToolsVer = '26.0.2'
```
Comment (emartynov) [inline build.gradle]: 26.0.3?

File: build.gradle [hunk 2]
```diff
@@ -27,7 +27,7 @@ buildscript {
     dependencies {
         // TODO: Please revert to a stable gradle version before releasing on prod
         // Alpha is required in devel to use Instant Run and speed up development process
-        classpath 'com.android.tools.build:gradle:2.3.1'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```
Comment (emartynov) [inline build.gradle]: You should use `google()` repository for this dependency. They don't distribute through the jcenter anymore.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Mon Mar 20 12:26:00 CET 2017
+#Sat Nov 25 17:06:20 MSK 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (emartynov) [inline gradle/wrapper/gradle-wrapper.properties]: 4.3.1?

**Review by emartynov — CHANGES_REQUESTED** [review]
Thanks for the contribution! Could you react to comments?

File: .travis.yml [hunk 4]
```diff
@@ -6,7 +6,9 @@ jdk:
 android:
   components:
     - tools
-    - build-tools-25.0.1
+    - platform-tools
+    - tools
+    - build-tools-27.0.1
     - android-25
```
Comment (ghost) [inline .travis.yml]: Should be `android-27`, as you updated `compileSdkVersion` to `27` and `buildtools` to `27.0.1`. Thanks

**Review by ghost — COMMENTED** [review]
In `.travis.yml` you forgot to change `android-25` to `android-27`. Please fix it

**Pull request closed without merging**